examples/network_graph/README.md

```diff
@@ -0,0 +1,36 @@
+# Network Node Classification Example
```

Comment:
LGTM!!

Reply:
If you can cite each dataset, please cite.

Reply:
fixed!